app/models/chargeback_rate_detail.rb

```diff
@@ -36,9 +67,12 @@ def find_rate(value)
 
   def cost(value)
     return 0.0 unless self.enabled?
-    value = 1 if group == 'fixed'
+
+    value = 1.0 if fixed?
+
     (fixed_rate, variable_rate) = find_rate(value)
-    hourly(fixed_rate) + hourly(variable_rate) * value
+
+    hourly(fixed_rate) +  hourly(variable_rate) * value
```

Comment:
thanks @gtanzillo for finding it -  I made change in separated commit 
https://github.com/ManageIQ/manageiq/pull/11648/commits/0b6e9d60fdb249c4f99815270bebcb5b6bb23bf9

I did it in slightly different way to see that only hourly **variable** part is adjusted regard to default unit metric in MetricRollup table.